cmd/atlas/mux.go

```diff
@@ -17,9 +21,19 @@ type (
 	// Driver implements the Atlas interface.
 	Driver struct {
 		*sql.DB
-		Differ    schema.Differ
-		Execer    schema.Execer
-		Inspector schema.Inspector
+		Differ      schema.Differ
+		Execer      schema.Execer
+		Inspector   schema.Inspector
+		MarshalSpec func(v interface{}, marshaler schemaspec.Marshaler) ([]byte, error)
+	}
+
+	schemaMarshal struct {
+		driver    *Driver
```

Comment:
printer? 
also dont need access to the full driver (coupling) use the MarshalSpec method directly

Reply:
Done